forte/data/data_store.py

```diff
@@ -160,9 +161,16 @@ def __init__(
         ``type_name``, their parent entry, and the order of corresponding attributes.
         The keys are fully qualified names of every type; The value is a
         dictionary with two keys. Key ``attribute`` provides an inner dictionary
-        with all valid attributes for this type and the indices of attributes
-        among these lists. Key ``parent_class`` is a string representing the
-        ancestors of this type.
+        with all valid attributes for this type and the information of attributes
```

Comment:
Can we restructure this docstring as bullet points of numbered lists? A bit hard to follow right now.